spec/features/course/assessment/submission/worksheet_spec.rb

```diff
@@ -78,6 +78,28 @@
         end
       end
 
+      scenario 'I can edit my existing comment', js: true do
```

Comment:
Actually do you think it is a good idea to combine `I can comment on answers`, `edit comment` and `delete comment` together? The JS tests takes time run and we should try to keep our build time as low as we can.